metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "simp-autofs",
-  "version": "6.0.2",
+  "version": "6.0.3",
```

Comment:
Code looks good, but the addition of package ensures to the top level api should be a minor bump I think